docs-src/concepts/what-is-a-payload-converter.md

```diff
@@ -0,0 +1,30 @@
+---
+id: what-is-a-payload-converter
+title: What is a Payload Converter?
+sidebar_label: Payload Converter
+description: A Payload Converter serializes data, converting objects or values to bytes and back.
+tags:
+  - term
+  - explanation
+---
+
+A Payload Converter serializes data, converting values to bytes and back.
+
+When you request a Workflow Execution through your Client and pass a data input, the input is serialized using a Data Converter that runs it through a set of Payload Converters.
+When your Workflow Execution starts, this data input is deserialized and passed as input to your Workflow.
+
+See the API reference for more information.
+
+- [Go](https://pkg.go.dev/go.temporal.io/sdk@v1.20.0/converter#PayloadConverter)
```

Comment:
All other languages point to the latest docs. If it's what we prefer, this url should be
```
https://pkg.go.dev/go.temporal.io/sdk/converter#PayloadConverter
```

Reply:
yes! also checking other links to make sure m linking to the latest.